starter-code/database.js

```diff
@@ -133,27 +143,39 @@ class Database {
         // productName is the name of the product that we want to buy
         // Think if you may need to implement two queries chained
         // remeber once it's finish to comment callback('Error buying product');
-        
-        callback('Error buying product');
+        callback(
+          null,
+          database
+            .collection("users")
+            .find("userFirstName")
```

Comment:
The `find` method expects a query object but you gave a string. The solution is `.find({firstName: userFirstName})`.